packages/system/kubeovn-plunger/Makefile

```
@@ -0,0 +1,19 @@
+export NAME=kubeovn-plunger
+export NAMESPACE=cozy-kubeovn
+
+include ../../../scripts/common-envs.mk
+include ../../../scripts/package.mk
+
+image:
```

checkmake (0.2.2)

[warning] Target body for "image" exceeds allowed length of 5 (12).

(maxbodylength)